ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,67 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function EinExprs.einexpr(tn::ITensorNetwork; optimizer::EinExprs.Optimizer)
+  IndexType = Any
+  T = vertextype(tn)
+
+  tensors = EinExpr{IndexType}[]
```

Comment:
I think it is confusing using the variable name `tensor` below to refer to an ITensor, but then `tensors` here is a collection of `EinExpr`. Maybe this could be called `einexprs`?

Reply:
I usually just call them `exprs`.

Reply:
Renamed them to `tensor_exprs`.